gcd/src/main/java/org/spine3/server/storage/datastore/DatastoreProperties.java

```diff
@@ -51,11 +54,27 @@
     private static final String EVENT_TYPE_PROPERTY_NAME = "event_type";
     private static final String EVENT_ID_PROPERTY_NAME = "event_id";
 
+    private static final String ARCHIVED_PROPERTY_NAME = "archived";
```

Comment:
Assuming the growing number of constants, I think we need to think of using `enum`s for them. Let's leave it as is for now because we need this update urgently. But let's sit and talk on making this part of the code better.